firmware/controllers/algo/engine2.cpp

```diff
@@ -174,17 +174,30 @@ void EngineState::periodicFastCallback(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	// Fuel cut-off isn't just 0 or 1, it can be tapered
 	fuelCutoffCorrection = getFuelCutOffCorrection(nowNt, rpm PASS_ENGINE_PARAMETER_SUFFIX);
 
-	// post-cranking fuel enrichment.
-	// for compatibility reasons, apply only if the factor is greater than zero (0.01 margin used)
-	if (engineConfiguration->postCrankingFactor > 0.01f) {
-		// convert to microsecs and then to seconds
+
+	    float afterStartEnrich;
```

Comment:
you don't need this separate block of declarations - you can declare and initialize in the same line.
example:
`float afterstartHoldTime = interpolate2d("aseHold", .....);`

Reply:
Yeah, now I remember why I did it like this.
It was to get the values in variables while debugging :-)

Reply:
You can still inspect the values in the debugger...  Remember that the variable doesn't exist until the line AFTER it's declared, so you have to step over a line to see what it did.